src/recursion.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function recursion(tree) {
+  const arrTree = [];
+  function recursionNode(node, array, currentDepth) {
+    if (!arrTree[currentDepth]) arrTree[currentDepth] = [];
+    arrTree[currentDepth].push(node.value);
+    if (node.left !== undefined) recursionNode(node.left, array, currentDepth + 1);
```

Comment:
`if (node.left !== undefined)` can be replaced with `if (node.left)` basically :)
In javascript, every expression passed as the parameter of the conditional structure is being interpreted as _boolean_ (implicit cast). And since every _object_ in JS is **truthy** your condition will work.